.mailmap

```diff
@@ -0,0 +1,17 @@
+# For documentation see: https://github.com/git/git/blob/master/Documentation/mailmap.txt
+# Full names and mail addresses are taken from the Github contributor profile page, or from 
+# https://api.github.com/users/xxxxx/events/public when the info does not show in the public profile page
+# Individual contributors should check and see if all their names/emails are aliased correctly
+
+Gadi Aleksandrowicz <gadial@gmail.com>
+Yael Ben-Haim <yaelbh@il.ibm.com>
+Shaohan Hu <hushaohan@gmail.com>
+Jay Gambetta <jay.gambetta@us.ibm.com>
+Shelly Garion <shelly@il.ibm.com>
+Ali Javadi-Abhari <ajavadia@users.noreply.github.com>
```

Comment:
We should get @ajavadia to comment here but he probably wants his email to map to his IBM email address om the git log instead of a github noreply address.

Reply:
yeah `ali.javadi@ibm.com` please